.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,59 @@
+name: PR Build
+
+on:
+    pull_request:
+        branches: [main]
+
+env:
+  IMAGE_TAG: latest
+
+jobs:
+    build-simplatform:
+        name: Building Simple Platform 
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+
+            - name: Set up JDK
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'semeru'
+                cache: maven
+
+            - name: Print Githash
+              run: |
+                echo $GITHUB_SHA > ./simplatform.githash
+              
+            - name: Building galasa-simplatform-application using maven
+              run: |
+               mvn -f galasa-simplatform-application/pom.xml deploy -Dgpg.skip=true \
+               -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/obr \
+               -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+               -Dgalasa.release.repo=file:${{ github.workspace }}/repo \
+               --batch-mode --errors --fail-at-end \
+               --settings ${{github.workspace}}/settings.xml
+              
+            - name: Building galasa-simbank-tests using maven
+              run: |
+                mvn -f galasa-simbank-tests/pom.xml deploy -Dgpg.skip=true \
+                -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/obr \
+                -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+                -Dgalasa.release.repo=file:${{ github.workspace }}/repo \
+                --batch-mode --errors --fail-at-end \
+                --settings ${{github.workspace}}/settings.xml
+            
+            - name: Build Simplatform image for testing
+              uses: docker/build-push-action@v5
+              with:
+                context: .
+                file: dockerfiles/dockerfile.simplatform
+                load: true
+                tags: simplatform:test

```

Comment:
This is actually for the simplatform maven artefacts (I appreciate thats not obvious from the tekton pipeline) so can we update this to simplatform-maven-artefacts to match convention

Reply:
done